index.bs

```diff
@@ -2862,6 +2862,8 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
    <code>browsingContext.PrintPageParameters</code> with the fields set to
    their default values.
 
+Note: The minimum page size is 1 point, which is (2.54 / 72) cm.
```

Comment:
This is just for documentation purposes. It's not strictly needed. It could be nice to link to https://www.w3.org/TR/css3-values/#absolute-lengths.

Reply:
Added the aforementioned link inline.